Source/server.js

```diff
@@ -9,6 +9,7 @@ var port     = process.env.PORT || 3000;
 var mongoose = require('mongoose');
 var passport = require('passport');
 var flash    = require('connect-flash');
+var fs       = require('fs'); // TEMP - for saving acquisuite POST data
```

Comment:
what is fs? node module?

Reply:
FS is a node module called "File System." It allows a node app to create/edit/delete files locally on the server. In this case, fs will create a file called "postData.txt" that'll store the contents of the POST data. Once we get some sample data and set up the database, fs will no longer be needed.

Reply:
Beautiful